pkg/keyring/keyring_darwin.go

```diff
@@ -141,7 +141,7 @@ func SetBearer(prefix, secret string) error {
 }
 
 func DeleteBearer(prefix string) error {
-	if err := deleteSecret(prefix, bearer); err != nil {
+	if err := deleteSecretKey(prefix, bearer); err != nil {
```

Comment:
I'm not sure this should be a fatal error here, since these keys does not always exists